web_client/dialogs/editStyleGroups.js

```diff
@@ -154,6 +154,7 @@ const EditStyleGroups = View.extend({
             if (oldid && this.collection.get(oldid)) {
                 this.model.set(this.collection.get(oldid).toJSON());
             }
+            this.collection.each((model) => { model.save(); });
```

Comment:
This saves groups that exist, but doesn't delete removed groups from local storage.

Reply:
Perhaps we can add `while (this.collection.length) { this.collection.at(0).destroy(); }` just before we reset the collection.